### PR TITLE
Upgrade to Unity 6 [MTT-9275]

### PR DESCRIPTION
**Pull request by FabianStoll**
### Description
This PR upgrades Boss Room to the newly released Unity 6 LTS, specifically version 6000.0.25f1.
It also includes an upgrade of the Netcode for GameObjects package to version 2.0.0.

With the release of Unity 6 the Multiplayer Play Mode package is now available to simulate up to four virtual players in the Editor for better multiplayer testing. We have replaced ParrelSync with Multiplayer Play Mode.

### Issue Number(s)
[MTT-9275](https://jira.unity3d.com/browse/MTT-9275)
[MTT-8706](https://jira.unity3d.com/browse/MTT-8706)

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink
 - [ ] An Index entry has been added in readme.md if applicable

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.coop?pullRequest=890) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.coop?pullRequest=890) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: RikuTheFuffs<br/>:x: FabianStoll<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.samples.coop?pullRequest=890) it.</sub>

File: Assets/DefaultVolumeProfile.asset.meta [hunk 1]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```
Comment (fernando-cortez) [inline Assets/DefaultVolumeProfile.asset.meta:1]: I believe this file is auto-generated, but we don't need it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,15 @@ Additional documentation and release notes are available at [Multiplayer Documen
...
 * Added a welcome dialog to guide users on their first time experience
 * Added a Table of Contents with helpful resources, that can be accessed from the `Tutorials > Show Tutorials` menu
+* Added the Multiplayer Play Mode package v1.3.1 to the project 
+
+### Changed
+* Upgraded Boss Room to Netcode for GameObjects v2.0.0
+* Upgraded editor version to 6000.0.25f1
+
+### Cleanup
+* Removed ParallelSync from the project
```
Comment (fernando-cortez) [inline CHANGELOG.md:20]:
```suggestion
* Removed ParrelSync from the project
```

File: ProjectSettings/Packages/com.unity.learn.iet-framework/Settings.json [hunk 3]
```diff
@@ -9,8 +9,8 @@
             {
                 "type": "System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089",
                 "key": "IET.DisplayWelcomeDialogOnStartup",
-                "value": "{\"m_Value\":true}"
+                "value": "{\"m_Value\":false}"
```
Comment (fernando-cortez) [inline ProjectSettings/Packages/com.unity.learn.iet-framework/Settings.json:12]: This should be reverted, so the tutorial popup is displayed on project open.
I believe when it's displayed, this field is being set back to false.

**Review by fernando-cortez — COMMENTED** [review]
Minor changes to make!
I had a few warnings when I opened the project for deprecated API. Those should probably be addressed with this upgrade -- I'll send those offline.

**Review by fernando-cortez — DISMISSED** [review]
Looks good to me!
I created a task to go through the rest of the warnings.
Let's wait to hear back from QA on the Android test.

**Review by fernando-cortez — APPROVED** [review]
Let's make a ticket to bump up the editor version when it's okay to do so.
Otherwise, good to go!

**Pull request merged**